packages/api/src/external/fhir/shared/json-validator.ts

```diff
@@ -36,7 +35,6 @@ export const validateFhirEntries = (bundle: Bundle): Bundle => {
       const resourceValidate = ajv.compile(getSubSchema(resourceType));
```

Comment:
I am wondering if the changes here will even be enough. Ajv is shared state too. Does calling ajv.compile share things across all resources too?

Reply:
Im concerned we just need to move the whole 
`const ajv = new Ajv({
  strict: false,
});

ajv.addMetaSchema(metaSchema);
const validate = ajv.compile(schema);`

into the function call